Document `txpool_content` JSON-RPC method (#1949)
hyperledger/besu-docs: docs/public-networks/reference/api/index.md

</TabItem>
</Tabs>

### `txpool_content`

Returns all pending (executable) and queued (non-executable) transactions in the pool, grouped by

> sender address and then by nonce.

Reviewer: ```suggestion
Returns all pending and queued transactions in the pool, grouped by
sender address and sorted by nonce.
```

#### Parameters

None

#### Returns

`result`: _object_ - transaction pool content object with the following fields:

- `pending`: _object_ - map of sender addresses to maps of nonce to [transaction objects](objects.md#transaction-object),

>   where the transactions are consecutive from the sender's current account nonce (executable)

Reviewer: ```suggestion
- `pending`: _object_ - map of sender addresses to maps of nonce to [transaction objects](objects.md#transaction-object),
  for transactions pending inclusion in the next block
```

> - `queued`: _object_ - map of sender addresses to maps of nonce to [transaction objects](objects.md#transaction-object),

Reviewer: ```suggestion
- `pending`: _object_ - map of sender addresses to maps of nonces to [transaction objects](objects.md#transaction-object),
  for transactions pending inclusion in the next block

- `queued`: _object_ - map of sender addresses to maps of nonces to [transaction objects](objects.md#transaction-object),
```

>   where the transactions have a nonce gap and cannot be executed yet (non-executable)

Reviewer: ```suggestion
  for transactions scheduled for future execution
```
